Formula/git-recent.rb

```diff
@@ -0,0 +1,16 @@
+class GitRecent < Formula
+  desc "See your latest local git branches, formatted real fancy"
+  homepage "https://github.com/paulirish/git-recent"
+  url "https://github.com/paulirish/git-recent/archive/v1.0.3.tar.gz"
+  version "1.0.3"
```

Comment:
Just version, yep. There is no line selection; it's generally implied that it's the line immediately above the comment FYI 👍